app/services/licences/fetch-licence-version-purpose-condition.service.js

```diff
@@ -10,45 +10,74 @@ const LicenceVersionPurposeCondtionModel = require('../../models/licence-version
 /**
  * Fetch the matching licence version purpose conditions and return data needed for the view licence page
  *
- * @param {string} id The UUID for the licence_version_purpose_id to fetch
+ * @param {object} ids The object contains the UUIDs for the licence_version_purpose_id and the purpose_id
  *
  * @returns {Promise<Object>} the data needed to populate the view licence page and some elements of the summary tab
  */
-async function go (id) {
-  if (!id) {
+async function go (ids) {
+  if (!ids) {
     return {
-      abstractionConditions: []
+      numberOfAbstractionConditions: 0,
+      uniqueAbstractionConditions: []
     }
   }
-  const LicenceVersionPurposeCondition = await _fetchLicenceVersionPurposeCondition(id)
-  const data = await _data(LicenceVersionPurposeCondition)
+
+  const licenceVersionIds = []
+
+  ids.forEach((item) => { licenceVersionIds.push(item.licenceVersionPurposeId) })
+
+  const LicenceVersionPurposeCondition = await _fetchLicenceVersionPurposeCondition(licenceVersionIds)
+  const data = await _data(LicenceVersionPurposeCondition, ids)
```

Comment:
Should have spotted this in previous PRs.

We don't normally capitilise variable names is the first thing. But also as we don't have a `limt(1).first()` in the query in fetch we'll be getting an array back.

The function we're calling has it right!

```suggestion
  const licenceVersionPurposeConditions = await _fetchLicenceVersionPurposeCondition(licenceVersionIds)
  const data = await _data(licenceVersionPurposeConditions, ids)
```